via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -27,43 +37,141 @@ export type TranscriptProps = {
    * the start of the video.
    */
   currentTime: number;
+
+  /**
+   * Callback invoked when the user selects a segment from the transcript.
+   */
+  onSelectSegment?: (segment: Segment) => void;
 };
 
 type TranscriptSegmentProps = {
   isCurrent: boolean;
+  onSelect: () => void;
   time: number;
   text: string;
 };
 
-function TranscriptSegment({ time, text }: TranscriptSegmentProps) {
-  // TODO - Use appropriate semantic elements here
+function TranscriptSegment({
+  isCurrent,
+  onSelect,
+  time,
+  text,
+}: TranscriptSegmentProps) {
   return (
-    <div>
-      <div>{time}</div>
-      <p>{text}</p>
-    </div>
+    <li
+      className={classnames(
+        'flex flex-row p-1 hover:text-black',
+        isCurrent && 'bg-grey-2',
+        !isCurrent && 'text-grey-6'
```

Comment:
A matter of personal preference, but I find objects easier to read when conditionally setting classes with `classnames`.

```suggestion
        {
           'bg-grey-2': isCurrent,
           'text-grey-6': !isCurrent
        }
```

Reply:
Lyza's code in eg. the pattern library seems to prefer the object style too. I'll make this change.